decidim-meetings/app/cells/decidim/meetings/dates_and_map/show.erb

```diff
@@ -19,7 +19,7 @@
 
   <%= static_map %>
 
-  <% if online? %>
+  <% if meeting.online? && meeting.iframe_access_level_allowed_for_user?(current_user) %>
```

Comment:
This should be as it was before your change as we're changing a section of the cell 

```suggestion
  <% online? %>
```